src/mds/agg_mds/redis_cache.py

```diff
@@ -86,10 +89,13 @@ async def get_all_named_commons_metadata(self, name: str):
         return await self.json_get(name, ".metadata")
 
     async def get_commons_metadata_guid(self, name: str, guid: str):
-        idx = await self.redis_cache.execute(
```

Comment:
I lied, I guess there is _one_ functional change. In my testing, the `metadata[{idx}]` does not work on our flavor of JSON redis. Does that work for you @craigrbarnes? Making this a two-phase operation got things working for me while testing.

Reply:
That works for me.